client/reader/comments/post-comment-list.jsx

```diff
@@ -129,35 +129,30 @@ class PostCommentList extends React.Component {
 		return commentIds.reduce( ( prevSum, commentId ) => prevSum + this.getCommentsCount( this.props.commentsTree.getIn( [ commentId, 'children' ] ) ) + 1, 0 );
 	}
 
+	/***
+	 * Gets comments for display
+	 * @param {Array<Number>} commentIds The top level commentIds to take from
+	 * @param {Number} take How many top level comments to take
+	 * @returns {Object} that has the displayed comments + total displayed count including children
+	 */
 	getDisplayedComments( commentIds, take ) {
```

Comment:
I wonder if this function should even exist now, given that it only calls take on the list basically.
But then I agree that it can make the calling code a bit unclear so might be worth keeping it